apps/wallet/src/ui/app/shared/page-main-layout/index.tsx

```diff
@@ -42,12 +52,30 @@ export default function PageMainLayout({
                         centerLogo && !topNavMenuEnabled && !dappStatusEnabled,
                 })}
             >
-                <Link to="/tokens" className="no-underline text-gray-90">
-                    <Logo networkName={networkName} />
-                </Link>
-                {dappStatusEnabled ? <DappStatus /> : null}
+                <div
+                    className={st.logoContainer}
+                    style={{ flexBasis: logoWidth }}
```

Comment:
Not a blocker, is it impossible to center without calculating the logo width?

Reply:
maybe, I've tried a few things though and seems calculating the logo width works better